src/EditorFeatures/Core.Wpf/LineSeparators/LineSeparatorTag.cs

```diff
@@ -34,7 +35,7 @@ public LineSeparatorTag(IEditorFormatMap editorFormatMap)
         /// <summary>
         /// Creates a very long line at the bottom of bounds.
         /// </summary>
-        public override GraphicsResult GetGraphics(IWpfTextView view, Geometry bounds)
+        public override GraphicsResult GetGraphics(IWpfTextView view, Geometry bounds, TextFormattingRunProperties format = null)
```

Comment:
i would not have this be optinoal.  that just makes it easy for the caller to forget to pass this along.

Reply:
I made it optional because otherwise it would be unused by the LineSeparatorTag, is that fine?

Reply:
that's fine.  in both cases it's passed in.  it's just not used.  Optional is about the experience hte caller has.